src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/help.html

```diff
@@ -1,3 +1,8 @@
 <div>
-    Triggers a new build for a given job.
+    <p>
+        Triggers a new build for a given job.
+    </p>
+    <p>
+        Use the <a href="https://www.jenkins.io/redirect/pipeline-snippet-generator">Pipeline Snippet Generator</a> to generate a sample pipeline script for the build step.
```

Comment:
Would be better to link it to the snippet generator itself instead of its docs

Reply:
Will it work to link to the snippet generator?  This page is included in both the online help inside Jenkins and the [Pipeline steps reference](https://www.jenkins.io/doc/pipeline/steps/pipeline-build-step/).  I assumed that a link to the snippet generator inside Jenkins would result in broken link complaints from readers of the Pipeline steps reference.